### DISCUSSION TITLE
Zahnputz-Timer mit Geschichten

### DISCUSSION THREAD

**Discussion by TopperDEL** [Wendelpunkt/ToypadChallenge#29] [Ideas]
Jedes Kind bekommt seine eigene Figur. Beginnt es mit dem Zähneputzen, stellt es seine Figur aufs Pad. Es beginnt ein Timer, der durch den Farbwechsel dargestellt wird (rot - zu kurz, orange - bald geschafft, grün/blinken - fertig). Während des Putzens kann dann eine Geschichte über einen Lautsprecher abgespielt werden. Entweder aus einer MP3-Sammlung oder vielleicht auch KI-generiert und damit jeden Abend neu.

Erweiterung: kommt die zweite Figur (rechtzeitig oder auch so) dazu, wird die Geschichte angepasst. Evtl. hat jede Figur andere Attribute (bei einem Jungen vielleicht "mag Autos" etc #Klischees) und diese kommen dann eher in der Geschichte vor. Will ich also als Einhorn-liebende Schwester nicht nur eine Geschichte über Autos hören, fange ich vielleicht zügiger mit dem Putzen an, weil dann meine Einhorn-Attribut-tragende Figur ebenfalls mit drauf steht.

Könnte bei meinen Kindern echt ein Zugpferd sein. :)

**Comment by tomwendel** [comment]
Wunderbare Idee. Wie wärs. wenn der Tag einfach direkt an der Zahnbürste befestigt wäre. Dann könnte man einen Handgriff sparen und der Trigger ist eben, wenn die Bürste angehoben wird.

**Reply by TopperDEL** [reply]
Die Zahnbürsten werden halt regelmäßig gewechselt (sind noch keine elektrischen) und daher müsste man das Tag auch ständig wechseln.
Aber diese Idee gefällt mir wirklich. Die wäre was für meine zwei.
